### DISCUSSION TITLE
Is there a way to copy save files to the IFS or a windows/Unix folder?

### DISCUSSION THREAD

**Discussion by a4ginatl** [codefori/vscode-ibmi#1360] [IBM i] [answered]
Does the tool have a option to copy SAVF's to or from a folder?

I am looking for a way other than to use FTP.

TIA, Darryl Freinkel.

**Comment by chrjorgensen** [comment, marked as answer]
@a4ginatl You can use the `scp` (copy over SSH) or `sftp` (FTP over SSH) commands - they're included with most SSH programs, including [Putty](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html).

**Reply by sebjulliand** [reply]
Once this extension will be published https://github.com/halcyon-tech/vscode-ibmi-fs (soon...), there will be actions to download and upload SAVF from/to libraries, too.

**Comment by a4ginatl** [comment]
Thank you. That worked.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562

**Comment by a4ginatl** [comment]
Thank you.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562
